### PR TITLE
feat-buttons

### PR DESCRIPTION
**Pull request by iamsaurav28**
Screenshot (63)

### CODE REVIEW

File: comp-layouts/button.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+     <title>button</title>
+</head>
+<body>
+
+     <div class="btn-hover box">
+          <button class="btn-1"> HOVER ME! </button>
+          <button class="btn-2"> AWESOME </button>
+      </div>
+   <div class=" btn-container box">
+      <button class="btn">Click Here</button>
```
Comment (shuntzhere) [inline comp-layouts/button.html]: Remove the extra space before the btn-container class. Also, indent the code properly.

File: stylesheets/abstract/variables.css [hunk 2]
```diff
@@ -3,4 +3,8 @@
   --red-alert: red;
   --light-green-alert: rgb(122, 248, 139);
   --green-alert: green;
+  --highlight: #f9e506;
+
+  --hover-bg: rgb(255, 0, 68);
+  --hover: #51fb42;
```
Comment (shuntzhere) [inline stylesheets/abstract/variables.css]: Try to use same type of variables, don't mix between solids, RGB and #.

**Review by shuntzhere — COMMENTED** [review]
Indent the code properly for better readability. Also, suggested a minor thing to update. Rest looks good to me.

File: comp-layouts/button.html (hunk 1 above)
Comment (iamsaurav28) [reply comp-layouts/button.html]: ok done

File: stylesheets/abstract/variables.css (hunk 2 above)
Comment (iamsaurav28) [reply stylesheets/abstract/variables.css]: ok done as u suggested

File: stylesheets/components/button.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+.btn-1:hover {
+  box-shadow: inset 300px 0 0 0 var(--highlight);
+  cursor: pointer;
+  color: var(--black);
+}
+/* second button */
+
+.btn-2 {
+  width: 200px;
```
Comment (shubhamkumar648) [inline stylesheets/components/button.css]: use rem units for better responsive

**Review by shuntzhere — APPROVED** [review]
It looks good now.

**Review by shubhamkumar648 — COMMENTED** [review]
use relative units for better performance on web page

File: stylesheets/components/button.css (hunk 3 above)
Comment (iamsaurav28) [reply stylesheets/components/button.css]: ok done as u suggested

Comment (shubhamkumar648) [reply stylesheets/components/button.css]: perfect,you can merge

**Review by shubhamkumar648 — APPROVED** [review]
remaining look good ! you can merge

**Pull request merged**